android/hello_sdl_android/src/main/java/com/sdl/hellosdlandroid/SdlService.java

```diff
@@ -450,4 +451,13 @@ public void onError(String error) {
 			sdlManager.getScreenManager().presentChoiceSet(choiceSet, InteractionMode.MANUAL_ONLY);
 		}
 	}
+
+	private Vector<TTSChunk> createSimpleTTSChunks(String simple) {
+		if (simple == null) {
+			return null;
+		}
+		Vector<TTSChunk> chunks = new Vector<>();
+		chunks.add(new TTSChunk(simple, SpeechCapabilities.TEXT));
+		return chunks;
+	}
```

Comment:
Can we remove this method and replace it with the proposed one liner
`Collections.singletonList(new TTSChunk(simple, SpeechCapabilities.TEXT));`